pom.xml

```diff
@@ -66,6 +66,11 @@
 			<artifactId>jaxb-api</artifactId>
 			<version>2.4.0-b180830.0359</version>
 		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
```

Comment:
Question: Why is this dependency needed?

Reply:
It is required to resolve the JAXB dependency. It was facing this [issue](https://stackoverflow.com/questions/51916221/javax-xml-bind-jaxbexception-implementation-of-jaxb-api-has-not-been-found-on-mo).